stations.csv

```diff
@@ -27310,7 +27310,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 29461;Pellizzano;pellizzano;;;46.3113983;10.7584273;;IT;Europe/Rome;t;f;f;t;f;f;;;f;;f;;f;u0pxky;t;;f;11418;t;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv29461;
 29462;Penia di Canazei;penia-di-canazei;;;46.4580433;11.7988164;;IT;Europe/Rome;t;f;f;t;f;f;;;f;;f;;f;;f;ITPEN;t;11168;t;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv29462;
 29463;Pera di Fassa;pera-di-fassa;;;46.4409089;11.6940573;;IT;Europe/Rome;t;f;f;t;f;f;;;f;;f;;f;;f;;f;11218;t;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv29463;
-29464;Pinzolo;pinzolo;;;46.1597701;10.7650269;;IT;Europe/Rome;t;f;f;t;f;f;;;f;;f;;f;u0pwsf;t;ITPIL;t;11458;t;;f;;f;;f;;f;8313846;t;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv29464;
+29464;Pinzolo;pinzolo;;;46.1597701;10.7650269;;IT;Europe/Rome;t;f;f;t;f;f;;;f;;f;;f;u0pwsf;t;ITGIT;t;11458;t;;f;;f;;f;;f;8313846;t;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv29464;
```

Comment:
I know it was the script that updated this, but I think it's wrong. According to Distribution data, `ITGIT` is a code for the city of Giustino and `ITPIL` is a code for the city of Pinzolo. So I think we need to rule out this change.